test/frontend/test_node.py

```diff
@@ -70,7 +70,11 @@ def do_test_expected(self):
     onnx_graph = convert_graph(tf_graph, output_node)
     onnx_model = helper.make_model(onnx_graph)
     backend_rep = prepare(onnx_model)
-    backend_output = backend_rep.run(onnx_feed_dict)[output_name]
+    backend_output = []
+    for ext_output in backend_rep.predict_net.external_output:
+      backend_output.append(backend_rep.run(onnx_feed_dict)[ext_output])
```

Comment:
Looks like if we have N outputs, the test will run N times.
Any better way to do this?